pkgs/os-specific/linux/r8125/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  pname = "r8125";
+  version = "9.003.02";
+
+  src = fetchFromGitHub {
```

Comment:
I'd add a similar disclaimer here as I did for r8168:
https://github.com/NixOS/nixpkgs/blob/8f43f0e4e8de346444bfd50e547e540dc8dee87f/pkgs/os-specific/linux/r8168/default.nix#L11
We shouldn't just trust the github repo blindly but at least verify that it matches the drivers from the realtek page.